zokrates_core/src/static_analysis/constrain_inputs.rs

```diff
@@ -0,0 +1,120 @@
+//! Add runtime boolean checks on user inputs
```

Comment:
That means we now check for all Boolean inputs strictly.

Eg in that example code:
```go
struct A {
        bool b
}

def main(A a) -> (field):
  field dummy = 12
  return dummy
```

we will add boolean checks for the input `a`.